website/content/docs/nia/release-notes/0-5-0.mdx

```diff
@@ -0,0 +1,356 @@
+---
+layout: docs
+page_title: CTS v0.5.0
+description: >-
+  Consul-Terraform-Sync release notes for v0.5.0
+---
+
+-> The release notes for v0.5.0 focus on a subset of configuration deprecations
+
+# Configuration Deprecations in v0.5
+
+We introduced a number of configuration deprecations in Consul-Terraform-Sync (CTS) v0.5. Many of these deprecations are part of an effort to streamline CTS task configuration and reduce the inconsistencies. As CTS support for more use-cases increased,  inconsistencies arose and required a lot of documentation to explain.
+
+This documentation outlines the deprecations and examples on how to upgrade your configuration for v0.5. The deprecated configurations will not be removed until v0.7 and later. We plan to also release a tool to automate upgrading your configurations to make the upgrade quicker and smoother.
+
+## Overview of Deprecations
+
+The deprecations address two major inconsistencies. First is the use of the terminology "source" in various configuration blocks and fields. The second is the exception to configuring a task condition or a task module input by using the `services` field and its related `service` block.
+
+### Address "source" terminology
+
+The word "source" is used in the name of a number of task configurations. "Source" in the configuration name commonly refers to the Terraform module though it is not immediately clear and requires additional documentation to explain.
+
+We deprecated all configuration with the term "source" and replaced it with "module" in the task configuration:
+
+| Deprecated Config | Replacement Config | Description |
+| --- | --- | --- |
+| `source` field | `module` field | The path to the Terraform module |
+| `source_input` block | `module_input` block | The configuration block for additional module input for the Terraform module |
+| `source_includes_var` field | `use_as_module_input` field | The field on a `condition` block to indicate whether the condition object should be used as module input |
```

Comment:
```suggestion
| `source_includes_var` field | `use_as_module_input` field | The field on a `condition` block to indicate whether the same values of the condition object should be used for the module input |
```
I was thinking it would be more clear if we say that it's the same values rather than the same object, but then we lose the "used as module input" in the description. 🤔  So feel free to disregard if you prefer what you have!

Reply:
I see what you're saying - it's inaccurate to say that its the same object. Let me know your thoughts on the below. Not sure if the "as" doesn't fit in place of the "for"

> The field on the `condition` block to indicate whether the values of the condition object should also be used as module input

Reply:
I like your wording here!